articles/Remote Debugging/how-to-remote-debugging-with-vorlonJS.md

```diff
@@ -0,0 +1,33 @@
+<!--
+   tags:
+       Debugging:
+           - "Remote debugging with VorlonJS"
+-->
+# Remote debugging with VorlonJS
+
+> This article explains how to remotely debugging and testing your JavaScript using `vorlon.js`. While working with a Mac and an iPad might be straight forward, achievieving the same thing with a PC is more complicate. Here is one possible approach.
+
+## Server (your local machine)
+
+Install VorlonJS server from npm
+```console
+$ npm i -g vorlon
+```
+
+Then, run the server
+```console
+$ vorlon
+$ The Vorlon server is running
+```
+
+The server is now running and you have the ability to listen to clients.
+To actually see the clients linked with Vorlon, you can go now to Vorlon dashboard by opening `http://localhost:1337/dashboard`
+
+## Client (remote device)
+To link your client page with the Vorlon server, you need to add the following script tag to your page. 
+
+```javascript
+<script src="http://YOUR_LOCAL_MACHINE:1337/vorlon.js"></script>
+````
+
+Open your client app and see the client appearing on the Vorlon dashboard.
```

Comment:
I watched the video on the Vorlon site and noticed there is also a proxy which runs by default. This could be very useful for us in quickly debugging frontend issues on our Spielplatz or production environments, without first setting it all up locally. Therefore I suggest including a short paragraph on that (after you have tested it on e.g. demo.taotesting.com).